### ISSUE TITLE
Problem with 0.6.2 on no_std

### ISSUE THREAD

**Issue by schungx** [tkaitchuck/aHash#67]
Seems like `0.6.2` keeps pulling in `std`.

Does not happen on `0.5.x`.

**Comment by tkaitchuck** [comment]
Do you have an error message? I don't see anything in my github actions tests. 
Could it be the `build.rs`? That was added in 0.6.

**Comment by schungx** [comment]
It says `lazy_static` brings in `std`...

**Comment by schungx** [comment]
```
   = note: the lang item is first defined in crate `std` (which `lazy_static` depends on)
   = note: first definition in `std` loaded from \\?\C:\Users\Stephen\.rustup\toolchains\nightly-x86_64-pc-windows-gnu\lib\rustlib\x86_64-pc-windows-gnu\lib\std-1be2ca690e46db46.dll, \\?\C:\Users\Stephen\.rustup\toolchains\nightly-x86_64-pc-windows-gnu\lib\rustlib\x86_64-pc-windows-gnu\lib\libstd-1be2ca690e46db46.rlib
   = note: second definition in the local crate (`no_std_test`)
```

**Comment by tkaitchuck** [comment]
Ok. That's probably a bug on their side then, because it did not used to. 
Nevertheless, I have released 0.6.3 which replaces lazy_static with once_cell, so it shouldn't be an issue.

**Comment by schungx** [comment]
`0.6.3` depends on `const-random` which depends on `lazy_static`...
